.ci/oci-devworkspace-happy-path.sh

```diff
@@ -42,8 +42,8 @@ deployChe() {
     -p openshift \
     --batch \
     --telemetry=off \
-    --installer=operator \
-    --workspace-engine=dev-workspace
+    --k8spodreadytimeout=240000 \
```

Comment:
I see that in che-operator we're using slightly longer timeout `--k8spodreadytimeout=6000000`

https://github.com/eclipse-che/che-operator/blob/d4a50d89668f6c08f56d9db627f996fd733971a5/build/scripts/minikube-tests/test-operator.sh#L36

Reply:
I see, I will update the PR